Module.yaml

```diff
@@ -0,0 +1,125 @@
+version: 1.0.0
+name: OPCUASubscribe
+description: OPC UA Subscribe connects to OPC UA Server and listens for incoming data messages and forwards the payload to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_OPCUA_input.png
```

Comment:
so it is type `process` module but you use icon for `input` (so it has a different colour). Can you ask Christina for `process` type icon for this module as it looks better in UI?

Reply:
requested

Reply:
module is changed to input based on our last call.